pkgs/applications/networking/cluster/k9s/default.nix

```diff
@@ -0,0 +1,25 @@
+with import <nixpkgs> {};
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, buildGoModule }:
```

Reply:
fixed

Reply:
`s/Package/Module/` and we should be good